libsrc/leddevice/schemas/schema-sk6812_ftdi.json

```diff
@@ -0,0 +1,59 @@
+{
+	"type": "object",
+	"required": true,
+	"properties": {
+		"output": {
+			"type": "string",
+			"title": "edt_dev_spec_outputPath_title",
+			"required": true,
+			"propertyOrder": 1
+		},
+		"rate": {
+			"type": "integer",
+			"format": "stepper",
```

Comment:
Remove the format:stepper line, as Hyperion is not yet on the latest JSON-Editor.
Otherwise the "stepping" does not work